release-please-config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "bootstrap-sha": "b0983ad4d9fcc9a612430a5bb4cdcea62c57ae25",
+  "last-release-sha": "b0983ad4d9fcc9a612430a5bb4cdcea62c57ae25",
```

Comment:
These two lines should be removed after the first release has been created with release-please. They're intended for release-please to know where to start from.